[BUG]: Running locally in Ubuntu 21.10 gives an error "cli.js not found"

### Describe the bug

Running locally in Ubuntu 21.10 generates an error regarding `cli.js`

```
$ make
Warning: Compiler Explorer is only tested against v14.x, but v17 was found.
Note that you might find issues if you decide to keep this version
Visit https://nodejs.org/ for installation instructions for the required version
To configure where we look for node, set NODE_DIR to its installation base
/usr/bin/npm install 
npm WARN deprecated har-validator@5.1.3: this library is no longer supported
npm WARN deprecated debug@3.2.6: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated querystring@0.2.0: The querystring API is considered Legacy. new code should use the URLSearchParams API instead.
npm WARN deprecated uuid@3.3.2: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated popper.js@1.16.1: You can find the new Popper v2 at @popperjs/core, this package is dedicated to the legacy v1
npm WARN deprecated vis@4.20.1: Please consider using https://github.com/visjs

> compiler-explorer@0.0.3 prepare
> husky install

husky - Git hooks installed

added 557 packages in 9s

32 packages are looking for funding
  run `npm fund` for details
rm -rf out/dist/static out/dist/manifest.json
./node_modules/webpack-cli/bin/cli.js "--node-env=production"
/bin/sh: 1: ./node_modules/webpack-cli/bin/cli.js: not found
make: *** [Makefile:57: webpack] Error 127
```

### Steps to reproduce

1. Install node in Ubuntu 21.10

```
cd ~
curl -fsSL https://deb.nodesource.com/setup_17.x | sudo -E bash -
```

2. Get Node.js and NPM installed by executing this command:

```
sudo apt-get install -y nodejs
```

```
$ node -v
v17.3.0
$ npm -v 
8.3.0
```

3. Clone

```
git clone git@github.com:compiler-explorer/compiler-explorer.git
cd compiler-explorer
make
```

### Expected behavior

Should install node packages and run server?

### Reproduction link

Not applicable

### Screenshots

Not applicable

### Operating System

Ubuntu Linux 21.10

### Browser version

_No response_

--- a/Makefile
+++ b/Makefile
@@ -35,7 +35,7 @@ prereqs: node_modules
 
 NODE_MODULES=.npm-updated
 $(NODE_MODULES): package.json | node-installed
-	$(NPM) install $(NPM_FLAGS)
+	NODE_ENV=development $(NPM) install $(NPM_FLAGS)
 	@rm -rf node_modules/.cache/esm/*
 	@touch $@
 

